go/service/main.go

```diff
@@ -490,3 +491,19 @@ func (d *Service) SimulateGregorCrashForTesting() {
 		d.G().Log.Warning("Can't simulate a gregor crash without a gregor")
 	}
 }
+
+func (d *Service) configurePath() {
+	defer d.G().Trace("Service#configurePath", func() error { return nil })()
+
+	var newDirs string
+	switch runtime.GOOS {
+	case "linux":
+		newDirs = "/usr/local/bin"
```

Comment:
We don't (currently!) have this $PATH problem on Linux, I think it's fine to just leave Linux out of it and keep the code simpler.